spec/features/post_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.feature 'Posts', type: :feature do
+  before :each do
+    @user_one = User.create(name: 'Nunes',
+                            photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQgGzZ0z42r96DLQES7kolpBPw5xM2UqS04GKKYyZiF&s', bio: 'Teacher from Brazil.')
+
+    @post_one = Post.create(author: @user_one, title: 'Hello', text: 'This is my first post')
+    @post_two = Post.create(author: @user_one, title: 'Title 2', text: 'This is my second post')
+
+    @comment_one = Comment.create(post: @post_one, author: @user_one, text: 'Hi Aron!')
+    @comment_two = Comment.create(post: @post_one, author: @user_one, text: 'Hi Karla!')
+  end
+
+  describe 'post/index' do
+    before(:each) do
+      visit "/users/#{@user_one.id}/posts"
+    end
+
+    it "I can see the user's profile picture." do
+      expect(page).to have_tag('img', src: @user_one.photo.to_s)
+    end
+
+    it "I can see the user's username." do
+      expect(page).to have_content(@user_one.name)
+    end
+
+    it 'I can see the number of posts the user has written.' do
+      expect(page).to have_content(@user_one.posts_counter)
+    end
+
+    it "I can see a post's title." do
+      expect(page).to have_content(@post_one.title)
+    end
+
+    it "I can see some of the post's body." do
+      expect(page).to have_content(@post_one.text)
+    end
+
+    it 'I can see the first comments on a post.' do
+      expect(page).to have_content(@comment_one.text)
+    end
+
+    it 'I can see how many comments a post has.' do
+      expect(page).to have_content(@post_one.comments_counter)
+    end
+
+    it 'I can see how many likes a post has.' do
+      expect(page).to have_content(@post_one.likes_counter)
+    end
+
+    it 'I can see a section for pagination if there are more posts than fit on the view.' do
+      expect(page).to have_tag('button')
```

Comment:
- [ ] Some of your spec test cases are failing. Kindly ensure that **all** tests are passing!

**Failing test case(s):**
![image](https://user-images.githubusercontent.com/12745474/200747428-29c27d2f-bb22-4875-9d5f-74f8a19d8c65.png)
![image](https://user-images.githubusercontent.com/12745474/200747565-f904b74b-1c87-48fe-9f39-942783d4b25d.png)

Reply:
✔️ Done. I did the two tests that are required in the project requirements. The other ones are not for this branch.
![image](https://user-images.githubusercontent.com/47539402/200859810-97452fe3-6b99-4f1f-b6cd-a2a01be47aa3.png)